package.json

```diff
@@ -10,7 +10,7 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
-    "prestart:dev": "docker-compose up -d db",
+    "prestart:dev": "docker-compose up -d db rabbitmq",
     "start:dev": "NODE_ENV=development nest start --watch",
```

Comment:
you can remove the app application/service from docker-compose file and then here do `docker-compose up - d` without specifying any service. so it will run all of them.

Reply:
Done